### ISSUE TITLE
npm versions != GIT tags

### ISSUE THREAD

**Issue by marc-mabe** [wix-incubator/redux-saga-tester#67]
Hi,

You release a new version on npm every day! Even thought there are no changes in this repo.

We are using @dependabot this this opens a new upgrade PR every day which is very annoying.

**Comment by daveisfera** [comment]
Yes, this is kind of confusing and definitely annoying, so it would be good if it was stopped
